modules/core/client/controllers/app.client.controller.js

```diff
@@ -163,6 +163,17 @@
        */
       $scope.$on('$stateChangeStart', function (event, toState, toParams) {
 
+        if (toState.requiresRole) {
+          if (!Authentication.user) {
+            toState.requiresAuth = true;
+          }
+          // Check if user has the required role
+          else if (Authentication.user && !Authentication.user.roles.includes(toState.requiresRole)) {
+            event.preventDefault();
+            $state.go('volunteering');
```

Comment:
We could use `goHome()` helper here and ideally show a message, perhaps just with `$window.alert('Hello?')` since this is only for admin stuff?

Reply:
What if we keep the 301 to volunteering in combination with this? `$window.alert('Wanna help us build Trustroots?');`

Reply:
"No access" message seems an odd place to try to activate folks to volunteer... 🤔 

Those potential volunteers should never end up to these routes — if someone who knows about these routes ends up to them but is denied access, we should clearly inform about that so that they know how to fix it. Otherwise it just seems broken.

Reply:
Most devs will not have admin access, yet quite a few will be testing /admin stuff on local machines.